### ISSUE TITLE
RichText: quill:toolbar error in a SPFx extension / Dialog

### ISSUE THREAD

**Issue by Louren** [SharePoint/sp-dev-fx-controls-react#494]
#### Category
[ ] Enhancement

[X] Bug

[ ] Question

#### Version
Please specify what version of the library you are using: [         ]

_If you are not using the latest release, please update and see if the issue is resolved before submitting an issue._

#### Expected / Desired Behavior / Question
We're using the RichText control in a Dialog in a SharePoint ListView command set to allow editing of list items in a customized way.

#### Observed Behavior
Upon loading, the console error shows op:
`quill:toolbar Container required for toolbar`

#### Steps to Reproduce
* Load and use <RichText /> in a dialog.

I can imagine this control wasn't developed with dialogs in mind (al references I see are from a webpart). 

The error seems to be that Quill doesn't know where to load the toolbar. I can see that an element is (being) created (refer to screenshot below, `#toolbar_9f90c055-9974-4327-8b32-c4ccb72785a4`, but that it apparently can't be found (this is also according to quill source). Indeed, I can't find the element in the DOM. 

![image](https://user-images.githubusercontent.com/4624387/75546241-6cdf8100-5a28-11ea-8427-15a22122c160.png)

Maybe relevant (and a difference between webparts and dialog-based plugins) is that the dialog is not yet shown (element not mounted). If this is indeed related to the error, than this might be a problem that other dialog-users will face as well?

I look forward to reading any suggestions!

**Comment by Louren** [comment]
Hello, is there anything we can do to help triage this issue?

**Comment by Katli95** [comment]
Yep, facing the same issue now when using in dialog (SPFx ListViewCommandSet Extension). Repro steps as original post, does anyone have any ideas as to what the issue might be?

**Comment by michaelmaillot** [comment]
Hi everyone,

I've tested the RichText in latest version (3.15.0) within a OUIFR Dialog and it actually works (display, update, right panel,...).

Can you confirm it on your side?

**Comment by Katli95** [comment]
I'm sorry I'm afraid I'm not working with SPFx religiously these days 😅 Props for pruning old tickets tho! ✌️
